demisto_sdk/commands/format/format_module.py

```diff
@@ -145,7 +145,9 @@ def format_manager(
     use_git = use_git or not input
 
     if input:
-        files = get_files_in_dir(input, supported_file_types)
+        files = []
+        for i in input.split(","):
+            files.extend(get_files_in_dir(i, supported_file_types))
```

Comment:
```suggestion
        files = list(itertools.chain((get_files_in_dir(part, supported_file_types) for part in input.split(","))
```

Reply:
having a mypy issue with this, will look later why